HashRegistrarSimplified.sol

```diff
@@ -231,6 +295,11 @@ contract Registrar {
         bid.setOwner(_owner);
         entry h = entries[_hash];
 
+        /* 
+        A penalty is applied for submitting pointless bids. This prevents bids which
+        are never intended to be revealed during the reveal period, which could otherwise
+        appear falsely as an expression of interest.
```

Comment:
ie. I reveal a bid, but another bid is waiting until the last second to reveal, I may be tempted to up my bid?